src/Components/EnergyCalculator/Steps/Disclaimer.tsx

```diff
@@ -28,7 +28,7 @@ const Disclaimer = () => {
   const navigate = useNavigate();
   // use defaults for the config on this page because the config won't be loaded
   // when the page is first rendered when coming from /select-state
-  const publicChargeOption = useConfig<{ link: string }>('public_charge_rule', { link: '' });
+  const publicChargeOption = useConfig<{ link: string; text: { props: { id: string; defaultMessage: string; } } }>('public_charge_rule', { link: '', text: { props: { id: '', defaultMessage: '' } } });
```

Comment:
Any config with the structure of `{"_label": "", "_default_message": ""}` is turned into a `FormattedMessage` automatically.
```suggestion
  const publicChargeOption = useConfig<{ link: string; text: FormattedMessageType}>('public_charge_rule', { link: '', text: <FormattedMessage id="landingPage.defaultPublicChargeLink" defaultMessage="Public Charge Rule" /> });
```